low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -67,3 +68,23 @@ def get_health_template(uid, is_fru: bool):
                 "specifics": []
             }
         }
+
+    @staticmethod
+    def set_health_data(health_data: dict, status, description=None,
+                        recommendation=None, specifics=None):
+        """Sets health attributes for a component."""
+        good_state = (status == "OK")
+        if not description:
+            description = "%s %s in good health." % (
+                health_data.get("uid"),
+                'is' if good_state else 'is not')
+        if not recommendation:
+            recommendation = 'None' if good_state\
+                else "Fault detected, please contact Seagate support."
```

Comment:
"please contact Seagate support." needs to be in a constant, as we may change it later to something more suitable. NOt sure which PR finally taking care of this but, one of them is having this constant defined, which should remain atlast

Reply:
Added DEFAULT_ALERT_RECOMMENDATION.